### DISCUSSION TITLE
22.1.11

### DISCUSSION THREAD

**Discussion by davidjgraph** [jgraph/drawio-desktop#1555] [General]
## Releases Notes for 22.1.11
[Windows Installer](https://github.com/jgraph/drawio-desktop/releases/download/v22.1.11/draw.io-22.1.11-windows-installer.exe)
[Windows No Installer](https://github.com/jgraph/drawio-desktop/releases/download/v22.1.11/draw.io-22.1.11-windows-no-installer.exe)
[macOS - Universal](https://github.com/jgraph/drawio-desktop/releases/download/v22.1.11/draw.io-universal-22.1.11.dmg)
Linux - [deb](https://github.com/jgraph/drawio-desktop/releases/download/v22.1.11/drawio-amd64-22.1.11.deb), [AppImage](https://github.com/jgraph/drawio-desktop/releases/download/v22.1.11/drawio-x86_64-22.1.11.AppImage) or [rpm](https://github.com/jgraph/drawio-desktop/releases/download/v22.1.11/drawio-x86_64-22.1.11.rpm)

Windows intel x32 releases are marked -ia32-

**ChangeLog:**

- Updates to [draw.io core 22.1.11](https://github.com/jgraph/drawio/blob/v22.1.11/ChangeLog). All changes from 22.1.3 to 22.1.11 are added in this build.

<hr /><em>This discussion was created from the release <a href='https://github.com/jgraph/drawio-desktop/releases/tag/v22.1.11'>22.1.11</a>.</em>

**Comment by shebeb** [comment]
how do I change the units from pt to mm now? Kinda liked the UI (in particular the settings tab) of 22.1.2 more.
Does anyone know how I can get the older version back as desktop app on mac? 
Thanks in advance!

**Reply by davidjgraph** [reply]
I don't understand from the description what it is you're saying has changed.

**Reply by shebeb** [reply]
How do I change on this version the measurement unit? e.g. From "pt" to "mm"

**Comment by AndorSebo** [comment]
FYI - This is probably the last desktop version that supports the macOS High Sierra (10.13.6.)
